deploy/02_sources.ts

```diff
@@ -242,13 +245,18 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
     // one hundred million tokens minted per network on testnets
     tokenAmount = BigNumber.from('100' + '000' + '000' + '000000000000000000');
     primaryNetwork = networks.ethereumTestnetGoerli;
+    if (environment == Environment.testnet) {
+      tokenAmount = BigNumber.from('10' + '000' + '000' + '000' + '000000000000000000');
+      targetChain = BigNumber.from(networks.ethereumTestnetGoerli.chain);
+      tokenRecipient = networks.ethereumTestnetGoerli.protocolMultisig;
+    }
   } else if (currentNetworkType == NetworkType.mainnet) {
     // ten billion tokens minted on ethereum on mainnet
     tokenAmount = BigNumber.from('10' + '000' + '000' + '000' + '000000000000000000');
     // target chain is restricted to ethereum, to prevent the minting of tokens on other chains
     targetChain = BigNumber.from(networks.ethereum.chain);
     // protocol multisig is the recipient
-    tokenRecipient = networks.ethereum.protocolMultisig;
+    tokenRecipient = '0xfC40b4233f8Ce60461e1D5FE50b3DDF0C50AE0b4'; //networks.ethereum.protocolMultisig;
```

Comment:
why is this hardcoded now instead of using `networks.ethereum.protocolMultisig;`

Reply:
Because this is the Holograph Research multisig. The protocolMultisig is for protocol management and is what is assigned as owner/admin of Holograph, but this one is hardcoded since it's the only multisig that will receive HLG on mainnet Ethereum network.

Reply:
responded below